dask/dask.yaml

```diff
@@ -7,12 +7,20 @@ tokenize:
   ensure-deterministic: false  # If true, tokenize will error instead of falling back to uuids
 
 dataframe:
+  backend:
+    library: null  # Backend dataframe library for input IO and data creation
+    allow-fallback: True  # Allow Dask to fall-back from the specified backend library
+    warn-fallback: True  # Raise a warning when Dask falls-back from the specified backend
```

Comment:
I'm wondering under what situations users would want this fallback to happen. Why not just error out? That seems potentially less confusing (though I see we're at least warning by default)

Reply:
The goal here is for the optional backend to never error out (but to optionally inform the user that the data is being moved from a different library). I think many cudf users are okay with the IO happening in pandas in the case that cudf doesn't support a specific IO format (as long as the data is moved into cudf automatically). You certainly get the best performance when cudf is used end-to-end, but I've seen plenty of cudf/dask_cudf user code that starts with cpu-specific code before moving to cudf.